elasticsearch_dsl/_async/search.py

```diff
@@ -113,6 +117,57 @@ async def delete(self):
             )
         )
 
+    @contextlib.asynccontextmanager
+    async def point_in_time(self, keep_alive="1m"):
+        """
+        Open a point in time (pit) that can be used across several searches.
+
+        This method implements a context manager that returns a search object
+        configured to operate within the created pit.
+
+        :arg keep_alive: the time to live for the point in time, renewed with each search request
+
+        The following example shows how to paginate through all the documents of an index::
```

Comment:
My thinking was that you may need to use point in time for other purposes besides iterating.

But I will remove the example here, which does not really make sense to ever use, since this is exactly what `iterate()` does.

Reply:
If you want two search over two indices, you can join them with a comma. OK for keeping it if you prefer.